### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by meghnasavit**
@meghnasavit

### CODE REVIEW

**Comment by meghnasavit** [comment]
pr comment @meghnasavit 
@aasiddiq 
@anuraag016 
,,,@shruti-shukla,,,

File: README.md [hunk 1]
```diff
@@ -2,3 +2,4 @@ This is a read me file.
 a
 
 bug bash test for @ mentions
+     vcvc
```
Comment (aasiddiq) [inline README.md:5]: @meghnasavit @anuraag016 

- [ ] @sushmita-m 
- [ ] @shruti-shukla 
- @apurva1112 
- @madhavan-v

**Comment by ashokirla** [comment]
adding a new comment 123wqdas @ashokirla @meghnasavit

File: README.md [hunk 2]
```diff
@@ -2,3 +2,4 @@ This is a read me file.
 a
 
 bug bash test for @ mentions
```
Comment (shruti-shukla) [inline README.md:4]: - Test 1 @aasiddiq 
- Test 2 @meghnasavit 
- **@shruti-shukla**

**Comment by shruti-shukla** [comment]
- Test 1 @aasiddiq 
- Test 2 @meghnasavit 
- **@shruti-shukla**

**Pull request closed without merging**